docs-ref-conceptual/use-cli-effectively.md

```diff
@@ -277,6 +277,47 @@ The following are Azure CLI environment variables:
 | **AZURE_CONFIG_DIR**           | The global directory for configuration files, logs, and telemetry. If unspecified, this value defaults to `~/.azure.` |
 | **AZURE_EXTENSION_DIR**        | The directory containing extension installations. If unspecified, this value defaults to the `cliextensions` subdirectory within the global configuration directory. |
 
+## Error handling for Azure CLI in PowerShell
+
+You can run Azure CLI commands in PowerShell, as described in [Choose the right Azure command-line tool](choose-the-right-azure-command-line-tool.md). If you do, be sure you understand Azure CLI error handling in PowerShell. In particular, Azure CLI doesn't create exceptions for PowerShell to catch.
+
+An alternative is to use the `$?` automatic variable. This variable contains the status of the most recent command. If the previous command fails, `$?` has the value of `$False`. For more information, see [about_Automatic_Variables](/powershell/module/microsoft.powershell.core/about/about_automatic_variables).
+
+The follow example shows how this automatic variable can work for error handling:
+
+```powershell
+az group create --name MyResourceGroup 
+    if ($? -eq $False) {
+        Write-Error "Error creating storage account"
+    }
+```
+
+The first command lacks the required `--location` parameter, so it fails. The conditional finds that `$?` is false and writes an error.
+
+You can use the `try` and `catch` keywords. Use `throw` to create an exception for the `try` block to catch:
```

Comment:
```suggestion
The `az` command fails because it is missing the required `--location` parameter. The conditional statement finds that `$?` is false and writes an error.

If you want to use the `try` and `catch` keywords, you can use `throw` to create an exception for the `try` block to catch:
```